### PR TITLE
Make Lyric viewer look a little better

### PR DESCRIPTION
**Pull request by brotatotes**

### CODE REVIEW

**Comment by brotatotes** [comment]
image

File: src/components/LyricView.tsx [hunk 1]
```diff
@@ -17,31 +24,47 @@ const LyricView: React.FC<LyricViewProps> = (props) => {
...
-      lyrics += `\n${versenumber}: `;
-      data["lyrics"][versenumber].forEach((line: String) => {
-        lyrics += `\t${line}\n`;
+      lyrics.push(<IonLabel key={key}>{getVerseText(versenumber)}</IonLabel>);
+      key++;
+      data["lyrics"][versenumber].forEach((line: string) => {
+        lyrics.push(
+          <IonItem key={key}>
+            <IonLabel className="ion-text-wrap">{line}</IonLabel>
+          </IonItem>
```
Comment (brandonxia01) [inline src/components/LyricView.tsx:51]: on light mode on my computer theres a line inbetween each line, is there a way to put each verse into an `IonLabel`, or somehow disable the line on the labels here?

Screen Shot 2020-11-19 at 9 48 20 PM

File: src/components/LyricView.tsx [hunk 2]
```diff
@@ -17,31 +24,47 @@ const LyricView: React.FC<LyricViewProps> = (props) => {
...
   }
+
+  function getVerseText(verse: string) {
+    return verse
+      .toLowerCase()
+      .replace("v", "Verse ")
+      .replace("c", "Chorus ")
+      .replace("b", "Bridge ")
+      .replace("p", "Pre-Chorus ");
+  }
```
Comment (brandonxia01) [inline src/components/LyricView.tsx:67]: Nice 👍

File: src/components/LyricView.tsx (hunk 1 above)
Comment (brotatotes) [reply src/components/LyricView.tsx:53]: Updated

**Review by brandonxia01 — APPROVED** [review]
Thanks for doing that!

**Pull request merged**